observer_pattern/weather_example/subject.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
+
+(function() {
+	var GLB = window.GLB = window.GLB || {}
+	console.log("before WEATHERSTATION_PROTOTYPE")
+
+	{console.log("hello")}
+
+	var WEATHERSTATION_PROTOTYPE = {
+		
+		registerObserver: function( observer ) {
+			console.log("hello prototype")
+			this.observers.push( observer );
+		},
+
+		removeObserver: function( observer ) {
+		// remove function from lodash.js
+			_.remove( this.observers, function( thing ) { 
+				return thing === observer 
+			});
+		},
+
+		notifyObservers: function ( observer ) {
+			var self = this
+			// this.observers.push({ update:function(x, y, z){
```

Comment:
don't need the observer placeholder anymore so it should be safe to delete this comment